### ISSUE TITLE
Translate OpenTelemetry system metrics

### ISSUE THREAD

**Issue by axw** [elastic/apm-server#5796]
Similar to what we did for JVM metrics (https://github.com/elastic/apm-server/issues/4919), we should translate [OpenTelemetry system metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md) to the equivalent metrics produced by Elastic APM agents, if possible, so they appear under the service overview metrics in Kibana.

**Comment by cyrille-leclerc** [comment]
@axw I love your initiative. Did you evaluate how we should prioritize the "official" system metrics versus the metrics captured by the [OpenTelemetry Collector receiver for Host Metrics](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/hostmetricsreceiver)?

I did some research when filling [Map the metrics of the OpenTelemetry Collector Host Metrics Receiver to ECS and store them in the metrics- indices #5310](https://github.com/elastic/apm-server/issues/5310) and it seemed that few of them were slightly different. 
I thought of prioritizing the integration of the [OpenTelemetry Collector receiver for Host Metrics](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/hostmetricsreceiver).

Maybe you have a different perspective .

**Comment by smith** [comment]
Related: #105923. We're not collecting the CPU and thread count metrics for JVM OpenTelemetry.

**Comment by axw** [comment]
@cyrille-leclerc no, I did not do any kind of evaluation - I just opened this for tracking.

**Comment by pjhavariotis** [comment]
Is there any development regarding this issue?

**Comment by ManojVeeranala** [comment]
Hi, Is there any update on this for dotnet?

**Comment by federicobarera** [comment]
Hi, we are trialling the integrations of dotnet apps via k8s auto instrumentation. I am wondering if this change will impact what we see here (currently nothing):

![image](https://github.com/elastic/apm-server/assets/6658124/752b3366-22dd-4e84-9173-eb8370bfc7bf)

**Comment by En3Tho** [comment]
Still nothing with 8.11.3 version. Only traces and logs show up.
